editor/editor_file_dialog.cpp

```diff
@@ -52,6 +52,15 @@ EditorFileDialog::RegisterFunc EditorFileDialog::unregister_func = nullptr;
 
 void EditorFileDialog::popup_file_dialog() {
 	popup_centered_clamped(Size2(1050, 700) * EDSCALE, 0.8);
+	focus_file_text();
+}
+
+void EditorFileDialog::focus_file_text() {
+	int lp = (file->get_text()).rfind(".");
```

Comment:
The extra parentheses are not necessary:
```suggestion
	int lp = file->get_text().rfind(".");
```